### PR TITLE
[SD] Add --vae_disable_img2col option to SD.

### PR DESCRIPTION
**Pull request by monorimet**
This PR adds the --vae_disable_img2col option to stable_args.

This slightly improves VRAM usage for users with limited vulkan device capabilities.

### CODE REVIEW

File: apps/stable_diffusion/src/utils/resources/opt_flags.json [hunk 1]
```diff
@@ -29,15 +29,21 @@
...
       },
       "fp32": {
         "default_compilation_flags": [],
         "specified_compilation_flags": {
           "cuda": [],
           "default_device": [
             "--iree-preprocessing-pass-pipeline=builtin.module(func.func(iree-flow-detach-elementwise-from-named-ops,iree-flow-convert-1x1-filter-conv2d-to-matmul,iree-preprocessing-convert-conv2d-to-img2col,iree-preprocessing-pad-linalg-ops{pad-size=16},iree-linalg-ext-convert-conv2d-to-winograd))"
+          ],
+          "default_lowmem": [
+            "--iree-preprocessing-pass-pipeline=builtin.module(func.func(iree-flow-detach-elementwise-from-named-ops,iree-flow-convert-1x1-filter-conv2d-to-matmul,iree-preprocessing-pad-linalg-ops{pad-size=32},iree-linalg-ext-convert-conv2d-to-winograd))"
```
Comment (powderluv) [inline apps/stable_diffusion/src/utils/resources/opt_flags.json:46]: Does winograd make sense without im2col ? @qedawkins @yzhang93

Comment (qedawkins) [reply apps/stable_diffusion/src/utils/resources/opt_flags.json:46]: yes, the two are mutually exclusive

File: apps/stable_diffusion/src/utils/resources/opt_flags.json [hunk 2]
```diff
@@ -29,15 +29,21 @@
           "cuda": [],
           "default_device": [
             "--iree-preprocessing-pass-pipeline=builtin.module(func.func(iree-flow-detach-elementwise-from-named-ops,iree-flow-convert-1x1-filter-conv2d-to-matmul,iree-preprocessing-convert-conv2d-to-img2col,iree-preprocessing-pad-linalg-ops{pad-size=32},iree-linalg-ext-convert-conv2d-to-winograd))"
+          ],
+          "default_lowmem": [
+            "--iree-preprocessing-pass-pipeline=builtin.module(func.func(iree-flow-detach-elementwise-from-named-ops,iree-flow-convert-1x1-filter-conv2d-to-matmul,iree-preprocessing-pad-linalg-ops{pad-size=32}))"
```
Comment (yzhang93) [inline apps/stable_diffusion/src/utils/resources/opt_flags.json:34]: Did you miss `iree-linalg-ext-convert-conv2d-to-winograd` flag? Or did you drop this intentionally?

**Review by yzhang93 — COMMENTED** [review]
We should probably add this option to untuned model as well, and revert this patch https://github.com/nod-ai/SHARK/commit/307f0334eed2e0589325f9adc53efa7c1e8af346

**Pull request closed without merging**